apistructs/cluster_dialer.go

```diff
@@ -45,6 +45,10 @@ func (c ClusterManagerClientType) String() string {
 	return string(c)
 }
 
+func (c ClusterManagerClientType) GenEventName() string {
```

Comment:
I think this is just the big scope of event.
for one client type, you maybe have multiple event name.

Reply:
adjusted, add event type